### PR TITLE
WPF-62566 Add the link to DocumentLanguage

### PR DESCRIPTION
**Pull request by Karthick-Arjunan**
### Description
Updated the UG document content for Supported Language page in SyntaxEditor.

### CODE REVIEW

File: wpf/Syntax-Editor/Language-Support/Supported-Languages.md [hunk 1]
```diff
@@ -13,7 +13,37 @@ Edit for WPF provides built-in support for a procedural and markup languages suc
 
 With the language support, EditControl enables the users to create, open, modify and save programming codes from different file types. EditControl provides built in Syntax highlighting and outlining support for all supported languages with SQL being exception in outlining support. It also provides built-in IntelliSense support for all procedural languages such as C# and Visual Basic.
 
-The `DocumentLanguage` property in the EditControl class enables the users to select the language. DocumentLanguage is a Language enum type property with default value as Text. The following lines of code can be used to change the DocumentLanguage property.
+The [`DocumentLanguage`](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Edit.EditControl.html#Syncfusion_Windows_Edit_EditControl_DocumentLanguage) property in the EditControl class enables the users to select the language. `DocumentLanguage` is an `enum` type property with default value as Text and includes the following languages:
+
```
Comment (Manivannan-E) [inline wpf/Syntax-Editor/Language-Support/Supported-Languages.md:17]: @Karthick-Arjunan, The [`DocumentLanguage`](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Edit.EditControl.html#Syncfusion_Windows_Edit_EditControl_DocumentLanguage) property in the EditControl class allows users to select their preferred language. This property is of `enum` type, with a default value of `Text`, and includes options for the following languages:

File: wpf/Syntax-Editor/Language-Support/Supported-Languages.md [hunk 2]
```diff
@@ -13,7 +13,37 @@ Edit for WPF provides built-in support for a procedural and markup languages suc
...
+11. VBScript
+
+12. Visual Basic
+
+13. XAML
+
+14. XML
+
+The following lines of code can be used to change the DocumentLanguage property.
```
Comment (Manivannan-E) [inline wpf/Syntax-Editor/Language-Support/Supported-Languages.md:46]: @Karthick-Arjunan  The following code snippet demonstrates how to change the `DocumentLanguage` property:

File: wpf/Syntax-Editor/Language-Support/Supported-Languages.md (hunk 1 above)
Comment (Karthick-Arjunan) [reply wpf/Syntax-Editor/Language-Support/Supported-Languages.md:17]: @Manivannan-E, Changed the content as mentioned.

File: wpf/Syntax-Editor/Language-Support/Supported-Languages.md (hunk 2 above)
Comment (Karthick-Arjunan) [reply wpf/Syntax-Editor/Language-Support/Supported-Languages.md]: @Manivannan-E, Changed the content as mentioned.

**Pull request merged**